docs/animation/intro.md

```diff
@@ -0,0 +1,30 @@
+Intro to the Animation API
+===============================
+
+The Forge Animation API lets you animate json (and B3D) models.
+Before you start reading this, you should know how vanilla JSON models are created, and should have 
+read the documentation on blockstates
+
+
+First, some terms:
+
+| Term | Description |
+| --- | --------- |
+| Model File | Your JSON file to be animated |
+| Armature File | The JSON file which defines the joints and clips of a model |
+| Clip | A collection of variables to apply to certain joints. Sort of an "animation group" |
+| Joint | A name for a json cube |
```

Comment:
Are you sure on that? That's a would be a problem, because forge supports loading of json, obj and b3d. So it should work on all 3. If not, we need to open another issue :-/.